osquery/core/query.cpp

```diff
@@ -25,6 +26,21 @@ namespace osquery {
 
 DECLARE_bool(decorations_top_level);
 
+QueryLogItem::QueryLogItem(std::string name_,
+                           std::string identifer_,
+                           ColumnNames columns_,
+                           size_t time_,
+                           uint64_t epoch_,
+                           std::string calendar_time_)
+    : name(name_),
+      identifier(identifer_),
+      time(time_),
+      epoch(epoch_),
+      calendar_time(calendar_time_),
+      columns(columns_) {
+  getDecorations(decorations);
```

Comment:
I'm not sure if it's a good idea. I think more logical place for executing parser was where it was before.